config/locales/models/motif_category.fr.yml

```diff
@@ -4,4 +4,4 @@ fr:
       motif_category: Catégorie de motifs
     attributes:
       motif_category:
-        participation_optional: RDV obligatoire
+        participation_optional: RDV facultatif
```

Comment:
C'est mieux obviously 👍 , mais  le nom de la colonne reste quand même ambigu (notamment avec la notion de rdv obligatoire dans les messages). Du coup il faudrait créer un ticket pour renommer cette colonne si on le fait pas là.

Reply:
je peux le faire là mais je trouve pas beaucoup mieux... `optional_rdv` serait ce qui colle le mieux à la traduction française que j'ai choisie

Reply:
Mais même `optional_rdv` peut dire que la présence au rdv est optionnel et non pas sa prise.
Peut-être que `optional_rdv_subscription` (`Prise de rdv facultative`) correspondrait à la réalité de cet attribut ?

Reply:
sounds good to me, j'ai fait la modif